cli/src/rust_template.rs

```diff
@@ -22,10 +21,19 @@ pub enum ProgramTemplate {
 }
 
 /// Create a program from the given name and template.
-pub fn create_program(name: &str, template: ProgramTemplate) -> Result<()> {
+pub fn create_program(
+    name: &str,
+    template: &ProgramTemplate,
+    tests_mod: Option<&str>,
```

Comment:
`tests_mod` doesn't really belong inside the logic of program creation — could you keep test related code in tests creation?